### DISCUSSION TITLE
Alerts for rule update or service errors

### DISCUSSION THREAD

**Discussion by IzacFrank** [Security-Onion-Solutions/securityonion#7159] [Unsupported Versions] [answered]
Hello all,

We would like to get alerts when "rule update" fails or service stops or fails. As far as I understand it is possible with Elastalert rule flatline type but I don't know what to check or where to start (We have working elastalerts for IDS detections already).

We have a standalone deployment with version 2.3.91.

Please guide me.

Thank you.
Isac

**Comment by dougburks** [comment, marked as answer]
If you want to use Elastalert, then you'll most likely need to come up with a process to send logs into Elasticsearch for whatever you're looking for.
